src/components/BaseLayerSwitcher/BaseLayerSwitcher.scss

```diff
@@ -0,0 +1,58 @@
+.rs-base-layer-switcher {
+  position: relative;
+  height: 80px;
+  width: 115px;
+  transition: 250ms width;
+  overflow: hidden;
+  display: flex;
+
+  &.rs-open {
+    width: 600px;
+  }
+
+  .rs-base-layer-switcher-button {
+    margin: 4px;
+    background-color: white;
+    border: 2px solid white;
+  }
+
+  .rs-layer {
+    position: relative;
+    min-width: 100px;
+    overflow: hidden;
+    background-color: gray;
+  }
+
+  .rs-close {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  .rs-open {
+    &:hover,
+    &.rs-active {
+      border: 2px solid red;
+    }
+  }
+
+  .rs-base-layer-switcher-title {
+    position: absolute;
+    width: 100%;
+    font-size: 10px;
+    padding: 2px 0;
+    background-color: rgba(255, 255, 255, 0.9);
+    text-align: center;
+    z-index: 2;
+
+    &.rs-closed {
+      color: white;
+      background-color: rgba(0, 0, 0, 0.7);
+    }
+  }
+
+  .rs-base-layer-switcher-image {
+    width: 100px;
+    height: 100px;
```

Comment:
why the height is 100px ? it goes outside the parnet div.

Reply:
img has been removed